.github/workflows/compliance.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         # debug
         ls -la
         git log  --pretty=oneline | head -n 10
-        ./scripts/ci/check_compliance.py -m Devicetree -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m Kconfig -c origin/${BASE_REF}..
+        ./scripts/ci/check_compliance.py -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m Kconfig -m DevicetreeBindings -c origin/${BASE_REF}..
```

Comment:
need to update L87 too

Reply:
> need to update L87 too

Now done and rebased; thanks for catching that.